### PR TITLE
system-variable: add tidb_enable_async_merge_global_stats

### PR DESCRIPTION
**Pull request by hawkingrei**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -3064,16 +3064,21 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
 - 默认值：`1`
 - 这个变量用于 TiDB analyze 分区表时，对分区表统计信息进行合并时的并发度。
 
+### `tidb_enable_async_merge_global_stats`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：布尔型
+- 默认值：`ON`, 从 v7.5.0 之前集群升级到 v7.5.0 之后版本时，该变量默认值为 `OFF`
```
Comment (hfxsd) [inline system-variables.md:3079]:
```suggestion
- 默认值：`ON`。从 v7.5.0 之前版本升级到 v7.5.0 及之后版本时，默认值为 `OFF`。
```

File: system-variables.md [hunk 2]
```diff
@@ -3064,16 +3064,21 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
+- 这个变量用于 TiDB merge global stats 阶段使用 async 方式进行统计信息合并, 可以有效避免 OOM 问题发生。
```
Comment (hfxsd) [inline system-variables.md:3080]:
```suggestion
- 这个变量用于 TiDB merge global stats 阶段使用 async 方式合并统计信息, 可以避免 OOM 问题发生。
```

**Review by qw4990 — APPROVED** [review]
LGTM

File: system-variables.md [hunk 3]
```diff
@@ -3064,16 +3064,21 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
 - 这个变量用于 TiDB analyze 分区表时，对分区表统计信息进行合并时的并发度。
 
+### `tidb_enable_async_merge_global_stats`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：布尔型
+- 默认值：`ON`。从 v7.5.0 之前版本升级到 v7.5.0 或之后版本时，默认值为 `OFF`。
+- 这个变量用于 TiDB 使用 async 方式合并统计信息, 可以避免 OOM 问题发生。
```
Comment (hfxsd) [inline system-variables.md:3080]:
```suggestion
- 这个变量用于 TiDB 使用异步方式合并统计信息, 以避免 OOM 问题。
```

**Review by hfxsd — APPROVED** [review]

File: system-variables.md [hunk 4]
```diff
@@ -3064,16 +3064,21 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
->
-> 当前版本中该变量控制的功能尚未完全生效，请保留默认值。
-
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
 - 默认值：`1`
 - 这个变量用于 TiDB analyze 分区表时，对分区表统计信息进行合并时的并发度。
 
+### `tidb_enable_async_merge_global_stats`
```
Comment (qiancai) [inline system-variables.md:3073]: 这是 7.5 新增的变量吗，如果是，可以加个标识 `<span class="version-mark">从 v7.5.0 版本开始引入</span>`

File: system-variables.md [hunk 5]
```diff
@@ -3064,16 +3064,21 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：布尔型
+- 默认值：`ON`。从 v7.5.0 之前版本升级到 v7.5.0 或之后版本时，默认值为 `OFF`。
+- 这个变量用于 TiDB 使用异步方式合并统计信息, 以避免 OOM 问题。
```
Comment (qiancai) [inline system-variables.md:3080]:
```suggestion
- 这个变量用于控制 TiDB  是否使用异步方式合并统计信息时, 以避免 OOM 问题。
```

File: system-variables.md [hunk 6]
```diff
@@ -3064,16 +3064,21 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
->
-> 当前版本中该变量控制的功能尚未完全生效，请保留默认值。
-
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
 - 默认值：`1`
 - 这个变量用于 TiDB analyze 分区表时，对分区表统计信息进行合并时的并发度。
 
+### `tidb_enable_async_merge_global_stats` <span class="version-mark">从 v7.5 版本开始引入</span>
```
Comment (hfxsd) [inline system-variables.md:3073]:
```suggestion
### `tidb_enable_async_merge_global_stats` <span class="version-mark">从 v7.5.0 版本开始引入</span>
```

File: system-variables.md [hunk 7]
```diff
@@ -3064,16 +3064,21 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
 - 这个变量用于 TiDB analyze 分区表时，对分区表统计信息进行合并时的并发度。
 
+### `tidb_enable_async_merge_global_stats` <span class="version-mark">从 v7.5.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 类型：布尔型
+- 默认值：`ON`。从 v7.5.0 之前版本升级到 v7.5.0 或之后版本时，默认值为 `OFF`。
+- 这个变量用于设置 TiDB 使用异步方式合并统计信息时, 以避免 OOM 问题。
```
Comment (qiancai) [inline system-variables.md:3080]:
```suggestion
- 这个变量用于设置 TiDB 使用异步方式合并统计信息, 以避免 OOM 问题。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**